Integrations/SignalWire API on Azure Function - Node.Js/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "signalwire-api-messaging",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "start": "func start",
```

Comment:
question: is `func` a package that needs to go in `devDependencies`?

Reply:
so in the readme.md users need to setup the Azure function environment which download an azure core tools that has the func executeable file that is used to test the function code